### ISSUE TITLE
Image-dump-related problems

### ISSUE THREAD

**Issue by chokladio** [zixaphir/appchan-x#473]
Ok, so, I've been having a lot of issues with the image dump in appchan x 2.4.1, both on chrome and firefox.

You're gonna have to bear with me now 'cause my english isn't flawless and there are several bugs here affecting each other causing my problems, but I'll do my best to give you a comprehendable description of the issue at hand.

Ok, let's make a list.

I'll begin with the "easiest to explain"-issue: appchan believes that some of my images are too large even though theese range between 80kb and 200kb. Trying to immideately post these images again does not work, however, the affected images have all been .jpg and converting them to .png does allow me to post them (effectively making the files larger....). 

So, after dumping a random amount of images, I get this "file too large" error, it can be after >150 images, it can be after 15, you never know.

Ok, now for the second issue, when I get the error message, it persists throughout a image dump, blocking EVERY picture that comes after with a "file too large" error. This can be solved by re-adding every picture in the list when they are up for posting (if I go through the list replacing, it will get stuck again, it only works if I replace the one image that's waiting to be posted).

So, after having these issues for a few days, appchan completely crashes and suddenly forgets all settings, and resets settings every time I go on to 4chan. Ok, no problem, others have had this issue before, so I re-install Chrome, delete the chromium folder, and re-install appchan... still the same issue.

I delete appchan from chrome, but I really want to be able to do image dumps on /e/, so I install appchan on firefox instead.

All seems well, so I begin dumping images, and after ~60 of them i get the "connection error, you may have been banned"-error message, fuck. 

So to sum it up, all was the same there, I get the message, the images still gets posted if I simply re-open them when they're about to be sent, but the message now persists for every image in the queue.

I really don't like "complaining" like this, but I don't know what to do anymore and living without appchan isn't really an option (at least since I haven't been able to find any good alternatives).

Thank you for your time and support

/chok
